airflow/www/static/js/dag.js

```diff
@@ -190,6 +190,8 @@ export function callModal(t, d, extraLinks, tryNumbers, sd) {
           cache: false,
           success(data) {
             externalLink.attr('href', data.url);
+            if(/^(?:[a-z]+:)?\/\//.test(data.url))
+              externalLink.attr('target', '_blank');
```

Comment:
```suggestion
            if (/^(?:[a-z]+:)?\/\//.test(data.url)) {
              externalLink.attr('target', '_blank');
            }
```

(Just to keep it consistent stylistically)

Reply:
Thanks for the review!  Change made in commit https://github.com/apache/airflow/pull/17477/commits/267eec89a1a0c764c85d302e5d9a2af424c6b3f6